homeworks/edgar_meloyan/Homework 4/ProductOfNumbers.java

```diff
@@ -0,0 +1,43 @@
+import java.util.Scanner;
+
+public class ProductOfNumbers{
+
+	public static void main(String[] args){
+
+		Scanner console = new Scanner(System.in);
+
+		System.out.print("Enter the first number ");
+		int num1 = console.nextInt();
+
+		System.out.print("Enter the second number ");
+		int num2 = console.nextInt();
+
+		System.out.print("Enter the third number ");
+		int num3 = console.nextInt();
+
+		System.out.print("Enter the fourth number ");
+		int num4 = console.nextInt();
+
+		System.out.print("Enter the fifth number ");
+		int num5 = console.nextInt();
+
+		System.out.print("Enter a natural number ");
+		int n = console.nextInt();
```

Comment:
you should use for and initialize array values directly.
for(int i = 0; i < n; i++)
array[i] = console.nextInt();